src/azure-cli-core/azure/cli/core/commands/constants.py

```diff
@@ -29,3 +30,6 @@
 DEFAULT_QUERY_TIME_RANGE = 3600000
 
 BLACKLISTED_MODS = ['context', 'shell', 'documentdb', 'component']
+
+SURVEY_PROMPT = Fore.YELLOW + Style.BRIGHT + 'Please let us know how we are doing: ' + Fore.BLUE \
```

Comment:
Starting with "Please" looks like a command. Maybe the prompt can be more friendly.

Reply:
IMHO, using "Please" is  kind of friendly.